src/mmw/js/src/modeling/tr55/quality/views.js

```diff
@@ -64,12 +69,28 @@ var ResultView = Marionette.LayoutView.extend({
 
 var TableRowView = Marionette.ItemView.extend({
     tagName: 'tr',
-    template: tableRowTmpl
+    template: tableRowTmpl,
+
+    templateHelpers: function() {
+        var load = this.model.get('load'),
+            runoff = this.model.get('runoff'),
+            adjustedRunoff = this.options.aoiVolumeModel.adjust(runoff);
+
+        return {
+            concentration: load / adjustedRunoff
```

Comment:
`runoff` is always null for me, which is leading to the `NaN` for concentration.